src/datetime/newInstance.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Returns a Date from its parts
+ * @param {number} year
+ * @param {number} month
+ * @param {number} day
+ * @param {number} hour
+ * @param {number} minutes
+ * @param {number} seconds
+ * @return Date
+ */
+
+import { DateTime } from "luxon";
+import { EMPTY_DATE_VALUE } from "../date/core";
+
+export const newInstance = (
+  year: number,
+  month: number,
+  day: number,
+  hour: number,
+  minutes: number,
+  seconds: number
+): Date => {
+  const ret = new Date(year, month - 1, day, hour, minutes, seconds, 0);
+  return ret.getFullYear() === year &&
```

Comment:
👀 interesting way of ensuring the provided date parts resulted in a valid date.

Reply:
Copied from date/newInstance.ts, credits to @dmendez :)